RELEASE/scripts/autoscend/iotms/mr2023.ash

```diff
@@ -989,3 +995,36 @@ void auto_useWardrobe()
 	// let maximizer handle if any of it is worth equipping
 	use($item[wardrobe-o-matic]);
 }
+
+boolean fightFlamingLeaflet() {
+	if(item_amount($item[inflammable leaf]) >= 11) {
+		auto_log_debug("summoning flaming leaflet (2389)");
+		leaves 11;
+
+		return true;
+	}
+
+	return false;
+}
+
+boolean fightFlamingMonstera() {
```

Comment:
Same for these functions. Might be easier to move all the common stuff to `auto_fightLeafMonster(monster m)`